[2025-02-13T06:07:46.744+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T06:07:46.772+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: create_table_postgres_dag.create_tables_raw_orders manual__2025-02-13T06:07:43.593039+00:00 [queued]>
[2025-02-13T06:07:46.786+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: create_table_postgres_dag.create_tables_raw_orders manual__2025-02-13T06:07:43.593039+00:00 [queued]>
[2025-02-13T06:07:46.787+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-13T06:07:46.808+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_tables_raw_orders> on 2025-02-13 06:07:43.593039+00:00
[2025-02-13T06:07:46.819+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2632) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-13T06:07:46.821+0000] {standard_task_runner.py:72} INFO - Started process 2642 to run task
[2025-02-13T06:07:46.820+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'create_table_***_dag', 'create_tables_raw_orders', 'manual__2025-02-13T06:07:43.593039+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/create_table_***_dag.py', '--cfg-path', '/tmp/tmp6hzybbbe']
[2025-02-13T06:07:46.824+0000] {standard_task_runner.py:105} INFO - Job 29: Subtask create_tables_raw_orders
[2025-02-13T06:07:46.844+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-13T06:07:46.889+0000] {task_command.py:467} INFO - Running <TaskInstance: create_table_postgres_dag.create_tables_raw_orders manual__2025-02-13T06:07:43.593039+00:00 [running]> on host 89bb8c6379b5
[2025-02-13T06:07:47.312+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/utils/log/secrets_masker.py:84 DeprecationWarning: The sensitive_variable_fields option in [admin] has been moved to the sensitive_var_conn_names option in [core] - the old setting has been used, but please update your config.
[2025-02-13T06:07:47.350+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='create_table_***_dag' AIRFLOW_CTX_TASK_ID='create_tables_raw_orders' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T06:07:43.593039+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-13T06:07:43.593039+00:00'
[2025-02-13T06:07:47.352+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-13T06:07:47.415+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-13T06:07:47.427+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-13T06:07:47.428+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=create_table_***_dag, task_id=create_tables_raw_orders, run_id=manual__2025-02-13T06:07:43.593039+00:00, execution_date=20250213T060743, start_date=20250213T060746, end_date=20250213T060747
[2025-02-13T06:07:47.479+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T06:07:47.490+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
